### DISCUSSION TITLE
Query data from store and receive at the same time

### DISCUSSION THREAD

**Discussion by Hackmeat** [thanos-io/thanos#6501] [Questions & Answers]
Hi,

i am trying to query the data of the receive (realtime data) and store data (historical data) in one query in the query frontend. But either one or the other is being queried but not both into one chart. Is there a possibitly to do so?

Running the Helmchart 12.6.2 / app version 0.31.0 
Single instance of all deployments

**Comment by satare** [comment]
Hi, do you have a 'global' querier ? 

This querier instance has 2 endpoints : your receiver and the store gateway, then the query-frontend query this querier.

```
query --endpoint=receiver.local --endpoint=storegateway.local
```

**Reply by Hackmeat** [reply]
query both at the samt time like this wasn't possbile

**Comment by Hackmeat** [comment]
I just found out that this was because i used diffrent kind of disks, so thanos wasnt able to put those together
